nym-connect/desktop/src-tauri/src/config/user_data.rs

```diff
@@ -14,6 +14,18 @@ pub enum PrivacyLevel {
     Medium,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct SelectedGateway {
+    address: Option<String>,
```

Comment:
Can probably remove the `Option<>` here, since one level higher up `SelectedGateway` is already stored as an `Option<>`?